src/pages/help-centre/data/_deposits-and-withdrawals.ts

```diff
@@ -0,0 +1,256 @@
+import { TQuestionsData } from './_data-types'
+import { deriv_app_url } from 'common/constants'
+
+const deposits_and_withdrawals: TQuestionsData = {
+    section: 'General',
+    category: '_t_Deposits and withdrawals_t_',
+    questions: [
+        {
+            question: '_t_What payment methods can I use for deposits and withdrawals?_t_',
+            category: 'Deposits and withdrawals',
+            sub_category: '_t_Payment methods_t_',
+            label: 'payment-methods',
+            answer: [
+                {
+                    translation_text:
+                        '_t_You can use debit and credit cards, e-wallets, cryptocurrency wallets, and payment agents for deposits and withdrawals (see our <0>payment methods page</0> for a complete list). Once you log in to your Deriv account, you’ll be able to see payment methods available in your country on the <1>Cashier page</1>._t_',
+                    translation_components: [
+                        { key: 0, type: 'link', to: '/payment-methods/' },
+                        { key: 1, type: 'link', to: 'https://app.deriv.com/cashier/' },
```

Comment:
Let's use the url variable from constant

Reply:
Will change all of the links in the help_center